trade-finance-manager-api/src/v1/controllers/acbs.controller.js

```diff
@@ -52,10 +57,17 @@ const addToACBSLog = async (payload) => {
 
 /**
  * Creates an ACBS task and adds it to the ACBS log.
- * @param {object} deal - The deal object containing the dealSnapshot property with the bank details.
+ * @param {String} deal - TFM deal object ID
```

Comment:
I think this type should be string and that the variable name should be dealId to match the function below

Reply:
I also think the return type should be `boolean`

Reply:
Good spot, have updated all.